src/miner/minerui.h

```diff
@@ -67,11 +67,18 @@ class GraphicsCardUI : public QWidget
 {
 	Q_OBJECT
 public:
+	enum class Connection {
```

Comment:
Connection is a bit too generic, use something else or prefix it with the Jah bs.